src/options.js

```diff
@@ -14,7 +14,7 @@ export let nonce = esmsInitOptions.nonce;
 if (!nonce) {
   const nonceElement = document.querySelector('script[nonce]');
   if (nonceElement)
-    nonce = nonceElement.getAttribute('nonce');
+    nonce = nonceElement.nonce;
```

Comment:
Perhaps:

```suggestion
    nonce = nonceElement.nonce || nonceElement.getAttribute('nonce');
```

will fix it if `.nonce` wasn't defined in older Firefox versions.

Reply:
That seems like a fair fix, but it still seems like the *Firefox Browser Test (89)* is failing. Do you know the reason why?

Reply:
Hmm, that test was passing before the fix. I'll rerun the CI, perhaps try it locally?

Reply:
Seems to work now! When will this fix be deployed?